data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,646 @@ mission "Remnant: Learn Sign Follow Up"
+			`	"One of our ideas has been that the anomaly is some sort of unstable wormhole. Reports of its existence go back to at least the times of the first Alpha War, with rumors that ships have entered Terminus to never return, presumably having entered the anomaly somehow. Now I'm not one to go off of rumors, but they certainly have to come from somewhere, so the idea was at least worth investigating.`
+			`	"Going under the assumption that it is a wormhole that is simply too unstable to allow a ship to pass through, we wanted to see if we could get anything at all to pass through. What we've discovered is that given a small and smooth enough object, we can get it to disappear within the anomaly without a trace." He then pulls out a small metallic sphere about the size of a ping pong ball from one of the cabinets which he holds onto by a wire.`
+			`	"Now this is only a model, but we created a number of small sensors about this size and sent them through the anomaly. We'd hoped that we could take some readings from whatever is on the other side of the anomaly, as whatever is entering it is clearly going somewhere, but we only ever get static coming back through the wire, followed by nothing a few minutes later. And unfortunately, whatever we send in we can't seem to pull back. The wire always snaps.`
+			`	"So the anomaly is clearly a wormhole. Now we just need to figure out how to get whatever we send into it to come back, and how to get something the size of a ship to enter it."`
+			`	"Sounds like you're making good progress," you respond. "Do you think I could get a copy of your research so far?"`
+			
+			label "terminus copy"
+			`	"A copy?" Ivan looks off into space for a moment. "Well, I suppose since you helped us from losing valuable equipment, I could make an exception for you. I typically don't share findings with those not on my research team until they're finalized. I've been burned one too many times because of that."`
+			`	Ivan moves to another filing cabinet and pulls out a number of folders. "This will be for your eyes only. Promise you won't share this with anyone."`
+			choice
+				`	"I promise."`
+					goto digitize
+				`	"I'll try."`
+			`	"Funny," Ivan says with a blank expression.`
+			label digitize
+			`	"Let me just get these folders digitized for you, then I can hand you a data chip with everything on it. It'll only be a few minutes."`
+			apply
+				set "remnant: got deep anomaly research"
+			`	Ivan leaves the room with all the folders, and when he returns he hands you a data chip before shoving all the folders back into the filing cabinet. "Well, anything else you need?"`
+			`	"No. That's all I was here for," you respond. "Good luck on the rest of your work."`
```

Comment:
All this sounds really good. That being said, this point right here seems like a great point to add another small branch in terms of trust. From my perspective, if First shows up, asks bluntly for data, and then leaves; that is suspicious to me. Or at least rude. Either way, something that is likely to reduce Ivan's trust.

On the other hand, if First chats with Ivan about his research and stuff, just happens to ask for a copy of the data in the middle, and then continues chatting some more about the research... That sounds like a natural part of the conversation and something that makes First feel more a part of the team.

Reply:
I'd rather not drag this on for too long. And there's really not much I can think to put after this point. Especially painful would be the face that this endpoint is condensed from various previous branches, meaning that to continue on from here would mean to add more branches to make sure we're talking about the right topic. Either that or we'd need some singular topic that works for any branch prior to this.